frontend/src/pages/Assessments/Quiz.tsx

```diff
@@ -0,0 +1,140 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import { connect } from "react-redux";
+import Layout from "../../components/Layout";
+import { getWords } from "./data/actions";
+import { getAllWords } from "./data/reducer";
+import styled from "styled-components";
+import PromptSpeech from "../../components/PromptSpeech";
+import PurpleButton from "../../components/PurpleButton";
+import { Word } from "../../models/types";
+import { CLOUD } from "../../constants/colors";
+
+interface IProps {
+  getWords: Function,
+  allWords: Array<Word>,
```

Comment:
Didn't know you could do it like this, I have been doing  Word[] :)

Reply:
Oh i think maybe doing Word[] is prob better lol im not super familiar with typescript rip ill chagne this